### PR TITLE
Add uiConfId for LiveDashboard on kmc.vars

### PR DESCRIPTION
**Pull request by gadyaari**

### CODE REVIEW

File: alpha/apps/kaltura/modules/kmc/lib/kmcUtils.class.php [hunk 1]
```diff
@@ -114,10 +114,10 @@ public static function getPartnersUiconfs($partner_id, $tag)
 		return $confs;
 	}
 
-	public static function getLiveAUiconf()
+	public static function getLiveUiconf($uiConfTag)
 	{
 		$c = new Criteria();
-		$tag = 'livea_player';
+		$tag = $uiConfTag;
```
Comment (yossipapi) [inline alpha/apps/kaltura/modules/kmc/lib/kmcUtils.class.php]: Use $uiConfTag as is why do you define another variable for it.

File: alpha/apps/kaltura/modules/kmc/lib/kmcUtils.class.php [hunk 2]
```diff
@@ -114,10 +114,10 @@ public static function getPartnersUiconfs($partner_id, $tag)
 		return $confs;
 	}
 
-	public static function getLiveAUiconf()
+	public static function getLiveUiconf($uiConfTag)
```
Comment (yossipapi) [inline alpha/apps/kaltura/modules/kmc/lib/kmcUtils.class.php]: I would change the name from getLiveUiconf to getLiveUiconfByTag

Comment (gadyaari) [reply alpha/apps/kaltura/modules/kmc/lib/kmcUtils.class.php]: ok

File: alpha/apps/kaltura/modules/kmc/lib/kmcUtils.class.php (hunk 1 above)
Comment (gadyaari) [reply alpha/apps/kaltura/modules/kmc/lib/kmcUtils.class.php]: I don't really know. didn't actually think about it

**Pull request merged**